Lombiq.HelpfulExtensions/FeatureIds.cs

```diff
@@ -13,4 +13,5 @@ public static class FeatureIds
     public const string Security = FeatureIdPrefix + nameof(Security);
     public const string TargetBlank = FeatureIdPrefix + nameof(TargetBlank);
     public const string SiteTexts = FeatureIdPrefix + nameof(SiteTexts);
+    public const string Workflows = FeatureIdPrefix + nameof(Workflows);
```

Comment:
I would suggest a new feature id since we have a feature that depends on an OC feature `OrchardCore.Users.ResetPassword` but possibly not all `Workflow` features will depend on it. 
See my comment for `Readme.md`.
```suggestion
    public const string Workflows = FeatureIdPrefix + nameof(Workflows);
    public const string ResetPasswordActivity = Workflows + "." + nameof(ResetPasswordActivity);
```

Reply:
Good point for this and above. I've named it ResetPassword though.